dynamic-plugins.default.yaml

```diff
@@ -471,6 +477,7 @@ plugins:
 
   # Techdocs
   - package: ./dynamic-plugins/dist/backstage-plugin-techdocs-backend-dynamic
+    disabled: false
```

Comment:
Why TechDocs being disabled again? This was explicitly enabled in 1.1 https://issues.redhat.com/browse/RHIDP-1299

Reply:
@kadel it's not disabled, the value is `false`

Reply:
🤦 I'm sorry. I just saw a line added and automatically assumed that it changes the original value without reading it 😇 . Sorry for noise